toolkit/data-sources/db-sync/src/bridge/mod.rs

```diff
@@ -117,7 +117,7 @@ fn utxo_to_transfer<RecipientAddress>(
 where
 	RecipientAddress: for<'a> TryFrom<&'a [u8]>,
 {
-	let token_delta = (utxo.tokens_out.0 as i128) - (utxo.tokens_in.0 as i128);
+	let token_delta = utxo.tokens_out.0 - utxo.tokens_in.0;
```

Comment:
For this one I would rather use `checked_sub`. This function is already `Option`.